src/server/scripts/Spells/spell_priest.cpp

```diff
@@ -794,6 +797,97 @@ class spell_pri_dark_indulgence : public SpellScript
     }
 };
 
+// 1215265 - Dispersing Light
+class spell_pri_dispersing_light : public AuraScript
+{
+    bool Validate(SpellInfo const* spellInfo) override
+    {
+        return ValidateSpellInfo({ SPELL_PRIEST_DISPERSING_LIGHT_HEAL })
+            && ValidateSpellEffect({ {spellInfo->Id, EFFECT_1} });
+    }
+
+    void HandleProc(AuraEffect* aurEff, ProcEventInfo& eventInfo)
+    {
+        HealInfo* healInfo = eventInfo.GetHealInfo();
+        if (!healInfo || !healInfo->GetHeal())
+            return;
+
+        Unit* caster = eventInfo.GetActor();
+        Unit* target = eventInfo.GetActionTarget();
+
+        if (!caster || !target)
```

Comment:
if event is not null, there must be an actor and a target of said event (action) by default

is any of these null at any point of the eventInfo or is there any possibility any of these mutated (this shouldn't be possible anyway)?